service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -358,35 +358,38 @@ public void deleteWorkspaceProperties(
         ActivityLogChangedTarget.WORKSPACE);
   }
 
-  /**
-   * Process the request to create a Azure cloud context
-   *
-   * @param workspace workspace in which to create the context
-   * @param jobId caller-supplied job id of the async job
-   * @param userRequest user authentication info
-   * @param resultPath optional endpoint where the result of the completed job can be retrieved
-   * @param azureContext deprecated azure context information, only used if BPM is not enabled
-   */
-  @Traced
-  public void createAzureCloudContext(
-      Workspace workspace,
-      String jobId,
+  public String cloneWorkspace(
```

Comment:
Moved cloneworkspace up

Reply:
can you add a trace annotation

Reply:
Added to all relevant functions in this class